invenio_communities/assets/semantic-ui/js/invenio_communities/api/invitations/api.js

```diff
@@ -0,0 +1,66 @@
+// This file is part of Invenio-communities
+// Copyright (C) 2022 CERN.
+//
+// Invenio-communities is free software; you can redistribute it and/or modify it
+// under the terms of the MIT License; see LICENSE file for more details.
+
+import { CommunityLinksExtractor } from "../CommunityLinksExtractor";
+import { bulkMembersSerializer } from "../serializers";
+import { http } from "../config";
+import _sample from "lodash/sample";
+
+export class CommunityInvitationsApi {
+  #urls;
+
+  constructor(community, LinksExtractor = CommunityLinksExtractor) {
+    this.#urls = new LinksExtractor(community);
+    this.community = community;
+  }
+
+  createInvite = async (members, role, message = undefined) => {
+    let payload;
+    if (message) {
+      payload = {
+        members: bulkMembersSerializer(members),
+        role: role,
+        message: message,
+      };
+    } else {
+      payload = {
+        members: bulkMembersSerializer(members),
+        role: role,
+      };
+    }
+
+    return await http.post(this.#urls.invitations, payload);
+  };
+
+  updateInvite = async (request_id, payload) => {
+    // TODO invitation service missing
+    const endpoint = `/api/requests/${request_id}`;
+    return await http.put(endpoint, payload);
```

Comment:
I think in that case the await is redundant as the user of this function will probably await and `try/catch` errors.

Reply:
as you said, it will probably use it like this, depending on the implementation of the user of this function. I left it consistent will all the previous implementations of the API methods, but if you want to get rid of the awaits here, I think we should change it everywhere, WDYT ?

Reply:
I think as a rule of thumb returning the await promise is redundant....but changing everywhere at once, might be not feasible at the moment. Anyway, I would handle my comment as suggestion so don't get blocked by that :)